developerGuide.t2t

```diff
@@ -228,6 +229,44 @@ Events will be covered in greater detail later.
 
 As with other examples in this guide, remember to delete the created app module when you are finished testing and then restart NVDA or reload plugins, so that original functionality is restored.
 
+++ App modules for hosted apps ++
+Some executables host various apps inside.
+These include javaw.exe for running various java programs and wwahost.exe for web apps in Windows 8 and later.
+
+If an app runs inside a host executable, the name of the app module must be the name as defined by the host executable, which can be found through AppModule.appName property.
+For example, an app module for a java app named "test" hosted inside javaw.exe must be named test.py.
+For apps hosted inside wwahost, not only the app module name must be the name of the loaded app, but the app module must subclass the app module class found in wwahost.
+
+Note that wwahost.exe hosts web applications or apps behaving as such.
+You should create an app module for apps hosted inside it if you believe the hosted app is not a true web application.
+This is the case for built-in apps such as Mail and Calendar in Windows 8 and 8.1.

```

Comment:
I know what you're trying to say here, but I don't think this paragraph will be understood by most people. First, the base wwahost app module (which they are subclassing) already disables browse mode. Second, "true web application" is a pretty hard thing to define. Arguably, any app that uses web technologies is a true web app. The question is more whether users want to interact with the content primarily as a document or primarily using the app's own keyboard support. I would suggest you just remove this paragraph altogether and let devs make the decision on their own. If they feel the need to write an app module for a wwahost app and are reading this section, they've already made the decision; this guidance isn't helpful at that point and might just confuse them.